<launch>
    <!-- Default arguments -->
    <arg name="drone_id1" default="falcon"/>
    <arg name="drone_id2" default="$(arg drone_id1)1"/> 
    <arg name="drone_name_1" default="falcon"/>
    <arg name="drone_name_2" default="$(arg drone_name_1)1"/>
    <arg name="target_x" default="2.0"/>
    <arg name="target_y" default="1.0"/>
    <arg name="target_z" default="1.0"/>
    <arg name="target_x1" default="2.0"/>
    <arg name="target_y1" default="-1.0"/>
    <arg name="target_z1" default="1.0"/>
    <!-- Node for Command-->
    <!-- Include the takeoff launch file -->
    <include file="$(find drone_controller)/launch/takeoff_2_drones.launch" />
    <!-- After takeoff, initiate the position control -->
    <group ns="BEP">
        <node name="Position_move_node"
              pkg="drone_controller"
              type="Position_move_node"
              output="screen">
            <param name="drone_id1" value="$(arg drone_id1)"/>
            <param name="drone_id2" value="$(arg drone_id2)"/>
            <param name="target_x" value="$(arg target_x)"/>
            <param name="target_y" value="$(arg target_y)"/>
            <param name="target_z" value="$(arg target_z)"/>
            <param name="target_x1" value="$(arg target_x1)"/>
            <param name="target_y1" value="$(arg target_y1)"/>
            <param name="target_z1" value="$(arg target_z1)"/>
        </node>
    </group>
</launch>

